src/utils/getProductImage.ts

```diff
@@ -107,4 +108,50 @@ const generateOptimizedImages = (
   return imageUrlArray;
 };
 
-export { generateOptimizedImages, getProductImageURLs };
+function getProductImagesFromQuery(item: Product) {
```

Comment:
Not a blocker: but we should probably change the name to `getProductImagesFromAttribute`, the current name sounds like you are running a new query

Reply:
Updated the function name